scripts/sql-scripts/create_tables.sql

```diff
@@ -20,5 +29,112 @@ CREATE TABLE IF NOT EXISTS `state_of_music`.`eventbrite_events` (
   `venue_name` VARCHAR(50) NULL,
   `venue_lat` DECIMAL(10,8) NULL,
   `venue_long` DECIMAL(11,8) NULL,
-  `genre` VARCHAR(50) NULL,
+  `genre` VARCHAR(50) NULL,  
+  `pop` DECIMAL(3,2) NULL,
+  `rock` DECIMAL(3,2) NULL,
+  `hip_hop` DECIMAL(3,2) NULL,
+  `rnb` DECIMAL(3,2) NULL,
+  `classical_and_jazz` DECIMAL(3,2) NULL,
+  `electronic` DECIMAL(3,2) NULL,
+  `country_and_folk` DECIMAL(3,2) NULL,
+  `state` VARCHAR(25) NULL,
+  `county` VARCHAR(50) NULL,
   PRIMARY KEY (`id`));
+
+CREATE TABLE IF NOT EXISTS `state_of_music`.`populations` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `sum_level` VARCHAR(10) NULL,
+  `state_code` INT(3) NULL,
+  `state_name` VARCHAR(50) NULL,
+  `state_abbr` VARCHAR(2) NULL,
+  `county_code` INT(4) NULL,
+  `county_name` VARCHAR(100) NULL,
+  `pop_2018` INT(10) NULL,
+  PRIMARY KEY (`id`));
+
+CREATE TABLE IF NOT EXISTS `state_of_music`.`all_events` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `source` VARCHAR(12) NOT NULL,
+  `source_id` VARCHAR(50) NOT NULL,
+  `venue` VARCHAR(100) NULL,
+  `venue_lat` DECIMAL(10,8) NULL,
+  `venue_long` DECIMAL(11,8) NULL,
+  `state` VARCHAR(25) NULL,
+  `county` VARCHAR(50) NULL,
+  `pop` DECIMAL(3,2) NULL,
+  `rock` DECIMAL(3,2) NULL,
+  `hip_hop` DECIMAL(3,2) NULL,
+  `rnb` DECIMAL(3,2) NULL,
+  `classical_and_jazz` DECIMAL(3,2) NULL,
+  `electronic` DECIMAL(3,2) NULL,
+  `country_and_folk` DECIMAL(3,2) NULL,
+  `dom_genre` VARCHAR(25) NULL,
```

Comment:
[FYI]
Gonna make this guy bigger because I just got a data too long for the dominant genre `rnb/classical_and_jazz/country_and_folk` hehe